content/docs/guide/gamemodes/climb.md

```diff
@@ -11,16 +11,92 @@ tags:
+Crouching will also limit your ablity to turn in the air and thus gain speed as well. This speed disadvantage is negated if you have already reached the Bhop speedcap on KZT since you will remain at 380 regardless of how much speed you try to gain.  
+
+**Standups**: While in the air, if you crouch close enough to the ground (TODO: get exact units) the engine will not let you uncrouch since you'll clip into the floor, instead it waits until the moment you've hit the ground or more space has opened up beneath you and uncrouches. This behavior gives you a nice grace period to execute a frame-perfect. (or tick-perfect) outcome. (Note that the faster you are falling, the shorter the grace period is).  
+
+**Standup Bhops**: Combines Standups with bhoping, the idea is that the best of both worlds is that you perfectly time and hold your crouch so that you only "tuck your legs" to get the extra distance. Sounds like a tall order but thanks to how "Standups" behave, to get the best outcome you only have to time the beginning of the crouch to be inside this window and the engine does the rest of the timing. This is not a replacement for regular Bhops or Crouch Bhops, since you may prefer to land sooner and jump sooner or need to aviod map hazards. Standup Bhops are only for squeezing extra distance out of a jump in an optimal manner. 
+
+**Double Duck [1.6 Climb only]**: In 1.6 Climb, while on the ground pressing duck and releasing it before you are fully on the ground will cause the game to unduck you prematurely, the result is that you pop 18 units off the ground (very small compared to a jump) you can improve this height by crouching again in the air resulting in being 36 units off the ground (better but still small compared to a jump).  
+you can use Double Ducks to climb small humps without wasting time overshooting with a jump, but in addition to this they have 1 big advantage over jumps:
+**ducking is not considered a jump so the jump-land penalty does not proc.**  
+This means you can gain speed with air strafing without worrying about slowing down upon landing. This is the used in Count Jumps.   
+
+**Count Jump [1.6 Climb only]**: Done by first gaining speed with a prestrafe than double ducking to gain more speed in the air, land then finally jump. It can also be used in a Bhop sequence where doing another jump would result in over/under shooting the next platform, in this case the double duck is used as a "mini-jump" in the sequence and most be timed just like a Bhop with the first tap of duck being right as you land and the second tap being right after. This is also sometimes called a **Count Jump**. 
+
+**Duck Roll [1.6 Climb only]**: Sometimes called Silent Walking, Russian Walking, or G-strafe in Counter-Strike, Duck Roll is its given name in Half-life so for consistency's sake It will be called Duck Roll.  
```

Comment:
> Sometimes called Silent Walking, Russian Walking, or G-strafe

g-strafe (ground strafe) is different from silent/russian walking. with silent/russian walking you do a standup after doing a doubleduck; giving you more airtime aka you do a doubleduck with your scrollwheel and then immediately after you hold your normal duck key (like ctrl) and release it before landing to do a standup. i think this may also be called standup g-strafe and it is used in maps for example to preserve speed on flat ground after coming off some big massive slide (used quite a lot in this video: https://www.youtube.com/watch?v=nKwA5Lib8uU )